File: ConexaoVagas Front/src/models/habilidadeCandidato.ts
import { Candidato } from "./candidato";
import { Habilidade } from "./habilidade";

export class HabilidadeCandidato {
    idHabilidadeCandidato?: number;
    idHabilidade?: number;
    idCandidato?: number;
    idCandidatoNavigation?: Candidato = undefined;
    idHabilidadeNavigation?: Habilidade = undefined;
}

File: ConexaoVagas API/Domains/HabilidadeCandidato.cs
using System;
using System.Collections.Generic;

namespace ConexaoVagasAPI.Domains
{
    public partial class HabilidadeCandidato
    {
        public int IdHabilidadeCandidato { get; set; }
        public int IdHabilidade { get; set; }
        public int IdCandidato { get; set; }

        public Candidato IdCandidatoNavigation { get; set; }
        public Habilidade IdHabilidadeNavigation { get; set; }
    }
}
